src/Identity/UI/src/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -30,6 +30,8 @@
     <Content Remove="@(Content)" />
     <Content Include="wwwroot\**\*" Pack="true" />
     <None Include="build\*" Pack="true" PackagePath="build\" />
+    <None Include="buildMultiTargeting\*" Pack="true" PackagePath="buildMultiTargeting\" />
```

Comment:
Question can you explain what is this needed for?  They both have the same targets include right?

Reply:
Its standard practice to do it. We can probably get rid of buildMultiTargeting, but it doesn't hurt either.

It is common practice to chain buildTransitive -> buildMultiTargeting -> build as it ensures the package works in all environments. The props file also works that way, but you don't see the files in the project because the static web assets tooling takes care of generating them when we are going to pack the final project, but if you look at the package you'll see them there.

Reply:
Gotcha, thanks for the explanation!